### PR TITLE
HPCC-17510 Version checking for platform dependency

### PR DESCRIPTION
**Pull request by Michael-Gardner**
Signed-off-by: Michael Gardner <michael.gardner@lexisnexis.com>

### CODE REVIEW

**Comment by Michael-Gardner** [comment]
Rodrigo, please review.

File: CMakeLists.txt [hunk 1]
```diff
@@ -139,6 +139,16 @@ IF (NOT "${CMAKE_VERSION}" VERSION_LESS "2.8.2")
         ENDIF ()
 
     ENDIF ( CMAKE_SYSTEM MATCHES Linux )
+    
+    IF ( "${packageManagement}" STREQUAL "DEB" )
```
Comment (rpastrana) [inline CMakeLists.txt]: Can we set the version as "${HPCC_MAJOR}.${HPCC_MINOR}.${HPCC_POINT}", then append the '~' if DEB?

**Review by rpastrana — COMMENTED** [review]
@Michael-Gardner one minor question

File: CMakeLists.txt [hunk 2]
```diff
@@ -139,6 +139,15 @@ IF (NOT "${CMAKE_VERSION}" VERSION_LESS "2.8.2")
         ENDIF ()
 
     ENDIF ( CMAKE_SYSTEM MATCHES Linux )
+
+    SET(HPCC_VERSION "${HPCC_MAJOR}.${HPCC_MINOR}.${HPCC_POINT}")
+    IF ( "${packageManagement}" STREQUAL "DEB" )
+        SET(HPCC_VERSION "${HPCC_VERSION}~")
```
Comment (Michael-Gardner) [inline CMakeLists.txt:145]: @rpastrana Made the change you requested.

**Comment by rpastrana** [comment]
@Michael-Gardner looks good.

**Comment by Michael-Gardner** [comment]
@richardkchapman Please merge

**Pull request merged**